static/src/javascripts/projects/common/modules/audio/AudioPlayer.js

```diff
@@ -335,6 +335,8 @@ export default class AudioPlayer extends Component<Props, State> {
 
         this.audio.addEventListener('volumechange', this.onVolumeChange);
         this.audio.addEventListener('timeupdate', this.onTimeUpdate);
+        // this should fire on Firefox
+        this.audio.addEventListener('ended', this.resetAudio);
```

Comment:
Will `resetAudio` fire twice in FF then? Once with the logic in `timeUpdate` and once when the `ended` event is dispatched.